.github/scripts/toolchain/group-gcc-test-failures.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+SCRIPT_DIRNAME=$(dirname ${BASH_SOURCE[0]})
+
+source $SCRIPT_DIRNAME/../config.sh
+
+TAG=$1
+GITHUB_STEP_SUMMARY=$2
+
+python3 $SCRIPT_DIRNAME/group-gcc-test-failures.py \
+    --dir $SCRIPT_DIRNAME/../testing-test-results \
```

Comment:
I'd rather use `$ARTIFACT_PATH` as a base directory for testing results. Furthermore, other steps handling the testing and test results are labeling the obtained test results with `$TAG` specified as a parameter. The `$TAG` can be, for example `baseline` a `changes` when doing comparative testing.